C++/Algorithms/Mathematical/SquareRoot.cpp

```diff
@@ -0,0 +1,51 @@
+#include <iostream>
+using namespace std;
+
+
+//These 2 functions will accept user input and return the resulting Square Root of that user input
+
+
+//Iterative Function
+double sqrtIterative(double x) {
+	double sqrt = x * pow(0.3, log10(x)); //Square Root Value
+	for (int i = 0; i < 5; i++)
+		sqrt = (sqrt + (x / sqrt)) * 0.5;
+	return sqrt;
+}
+
+
+//Recursive function
+double sqrtRecursive(double num, double prev)
+{
+	double next = (prev + num / prev) / 2;  //Smallest Square Root Value
+	if (fabs(next - prev) < DBL_EPSILON * next)
+		return next;
+	return sqrtRecursive(num, next);
+}
+
+int main(void) {
+
+
+
+	//Iterative Function
+
+	cout << "Iterative Function" << endl;
+	cout << "Please enter a number" << endl;
+	double iFn;
```

Comment:
Also, please have elaborate and meaningful variable declarations. Change the same wherever applicable.